mar/json/rpc/response.hoon

```diff
@@ -18,6 +19,16 @@
   ++  json                                              ::  from json
     =,  dejs-soft:format
     |=  a=json
+    ^-  raw-response
+    =;  resp
+      ?-    -.resp
+          %error   resp
+          %fail    resp
+          %batch   resp
+          %result
+        ~|  resp
+        [%result id.resp (crip (en-json:html res.resp))]
```

Comment:
The need to re-serialize here make me lean even harder towards just requesting `%httr` from %eyre.